Contrast-Security-OSS/vulneruby_engine
# RUBY-1065-Expand-App-Testing-Matrix

### Changes Introduced  :sparkles: 
This PR introduces GHCR and introduces the RUBY_VER var when building the app base image.

### How To Test :alembic:
1.) Check this out.
2.) Build the base and app image from something other than Ruby 2.7 using RUBY_VER
3.) 🤑 profit

### Additional Details :speech_balloon:
When building locally, will still use remote base image. Will need to modify file not to when building locally. Probably should implement an option somehow, but I'm not sure what options can be used in an if statement before the FROM in a dockerfile.
